com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1723,217 +1723,29 @@ private List<TaskResult> doBuild(IProgress monitor, String... commands) throws T
 
 
 
+
     @SuppressWarnings({ "rawtypes", "unchecked" })
     private List<TaskResult> runTasks(IProgress monitor) throws IOException, CompileExceptionError {
-        // set of all completed tasks. The set includes both task run
-        // in this session and task already completed (output already exists with correct signatures, see below)
-        // the set also contains failed tasks
-        Set<Task> completedTasks = new HashSet<>();
-
-        // the set of all output files generated
-        // in this or previous session
-        Set<IResource> completedOutputs = new HashSet<>();
 
-        List<TaskResult> result = new ArrayList<>();
+        TextureGenerator.maxThreads = getMaxCpuThreads();
```

Comment:
if we use threads for tasks, shouldn't we limit it for TextureGenerator then (or use only one)?

Reply:
Yes, maybe. This is the max threads used by basis right?

Reply:
I'm setting it to 1